src/main/scala/io/pravega/connectors/spark/PravegaMicroBatchStream.scala

```diff
@@ -129,4 +133,33 @@ class PravegaMicroBatchStream(
     s"PravegaMicroBatchStream{clientConfig=${clientConfig}, scopeName=${scopeName}, streamName=${streamName}" +
       s" startStreamCut=${startStreamCut}, endStreamCut=${endStreamCut}}"
   }
+
+  override def latestOffset(start: Offset, readLimit: ReadLimit): Offset =
```

Comment:
This method is formatted very strangely. Can you run it through the autoformatter?

Reply:
this has been taken care of.